server/http_test.go

```diff
@@ -316,9 +316,9 @@ func TestSubscribe(t *testing.T) {
 			api := &plugintest.API{}
 			p := Plugin{}
 
-			api.On("LogDebug", mockAnythingOfTypeBatch("string", 11)...).Return(nil)
-			api.On("LogError", mockAnythingOfTypeBatch("string", 10)...).Return(nil)
-			api.On("LogError", mockAnythingOfTypeBatch("string", 13)...).Return(nil)
+			api.On("LogDebug", mockAnythingOfTypeBatch("string", 11)...).Return()
```

Comment:
why it was nil before and now it's not? It doesn't seem like a needed change, but I might be wrong.

Reply:
@javaguirre The function `LogError` and `LogDebug` does not have a return type.